apps/api/src/domain/models/audio-item/entities/audio-item.entity.ts

```diff
@@ -90,7 +92,8 @@ export class AudioItem<T extends CoscradText = string> extends Resource {
 
         this.lengthMilliseconds = lengthMilliseconds;
```

Comment:
We need a `dateRecorded` either here or on the media item model.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184431482) is a story for that.